api/net/Areas/Subscriber/Controllers/ReportController.cs

```diff
@@ -147,6 +148,11 @@ public IActionResult FindById(int id)
         var instances = _reportService.GetLatestInstances(id, user.Id);
         report.Instances.Clear();
         report.Instances.AddRange(instances);
+        if (instances.Any() && includeContent)
+        {
+            var instance = instances.First();
+            instance.ContentManyToMany.AddRange(_reportInstanceService.GetContentForInstance(instance.Id));
```

Comment:
To improve performance content is not automatically returned by a report instance, it needs to be fetched separately.